Update QALD 8 and 9 datasets

DnlKI opened the pull request.

johannes-menzel reviewed (commented):
In updated-qald-8-9-merged-multilingual.json and in updated-qald-9-train-multilingual.json by chance I found the following question strings: 
- "Welche Feiertage werden auf der Welt gefeiert? test1"
- "Which holidays are celebrated around the world? test2"
Could you remove these debug messages from the back? And if you remember any more such debug messages could you also remove them? 

Also if I recall correctly, we wanted to have qald8-train + qald8-test + qald9-train as the merged train dataset and qald9-test as the merged test dataset, or is that not possible because there is overlap between qald8-test and qald9-test?

DnlKI commented:
Good catch, I have removed the debug duplicate question and added a new dataset, which consists of all questions from qald8-train + qald8-test + qald9-train.
